src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -301,7 +331,11 @@ const SkillsQuizStepper = ({ isStyleAutoSuggest }) => {
                   variant="outline-primary"
                   onClick={() => setCurrentStep(STEP3)}
                 >
-                  See more course recommendations
+                  <FormattedMessage
+                    id="enterprise.skills.quiz.v1.see.more.recommendations.label"
+                    defaultMessage="See more course recommendations"
+                    description="Label prompting the user to see more course recommendations on the skills quiz v1 page."
```

Comment:
Will this redirect the user to new page? can we add little bit more context in the description?